docs/molgenis/dev_beaconv2.md

```diff
@@ -84,14 +84,25 @@ Model endpoints for which record-level GET and POST requests are implemented are
   Data is retrieved from the _Individuals_ table.
 - `/filtering_terms` returns a list of the filtering terms accepted by that Beacon instance.
 
-GET request are record requests by default and therefor viewer permission on the data is needed to perform the queries.
-For post requests a
 
 ### Permissions
 
-By default, a new database will have VIEWER permission for all users including anonymous requests. How to set up
+By default, a new database will have **VIEWER** permission for all users including anonymous requests. How to set up
```

Comment:
no, by default a schema has no permission, am I right? So you need to set it.
Or does the fair data hub already set it?

Reply:
This is indeed the default for beacon data profiles. Made this explicit.